portfolio/src/main/webapp/script.js

```diff
@@ -46,47 +46,17 @@ function getComments() {
     commentsListElement.innerHTML = '';
 
     comments.map((comment) => {
-      commentsListElement.appendChild(createListElement(comment));
-      //commentsListElement.appendChild(createCommentSection(comment));
+      commentsListElement.appendChild(createCommentSection(comment));
     });
   });
 }
 
-/** Creates an <li> element containing text. */
-function createListElement(comment) {
-  const liElement = document.createElement('li');
-  liElement.innerText = "USERNAME: " + comment.username + " COMMENT: " + comment.commentBody;
-  return liElement;
-}
-
 function createCommentSection(comment) {
   const commentWrapper = document.createElement("div");
   commentWrapper.id = "comment-wrapper";
-
-  commentWrapper.innerHTML = '<div id="comment-header"> <p id="username-display"></p>\
-    <p id="timestamp-display"></p></div> <p id="comment-display"></p> <hr class="solid" />';
-  
-  const commentHeader = document.getElementById("comment-header");
-
-  const usernameDisplay = document.getElementById("username-display");
-  usernameDisplay.innerHTML = comment.username;
-  
-  const timestampDisplay = document.getElementById("timestamp-display");
-  timestampDisplay.innerHTML = comment.timestamp;
-
-  const commentDisplay = document.getElementById("comment-display");
-  commentDisplay.innerHTML = comment.commentBody;
-
-  const divider = document.createElement("hr");
-
-  // Append username and timestamp to comment header
-  commentHeader.appendChild(usernameDisplay);
-  commentHeader.appendChild(timestampDisplay);
-
-  // Append child nodes to comment wrapper
-  commentWrapper.appendChild(commentHeader);
-  commentWrapper.appendChild(commentDisplay);
-  commentWrapper.append(divide);
+  commentWrapper.innerHTML = `<div id="comment-header"> <p id="username-display">${comment.username}</p>\
```

Comment:
Setting innerHTML directly is not recommended because of performance issue: https://stackoverflow.com/questions/11515383/why-is-element-innerhtml-bad-code

Reply:
The stackoverflow talks about when we want to append the `innerHTML`, so it's modifying an old element. However, in my case, I'm creating a new element for each comment section. Is it the same thing? Do you have any suggestions on alternatives to achieve the same objective (creating a comment section with comment header and content)? Thanks!